website/components/DocLayout.js

```diff
@@ -354,7 +362,7 @@ export default function DocLayout({ children, toc, version, headings }) {
           href="https://cdnjs.cloudflare.com/ajax/libs/prism/1.15.0/themes/prism.min.css"
         />
         <link rel="stylesheet" href="/fonts/dank/dmvendor.css" />
-        <title>Fela - {currentPage}</title>
+        <title>{currentPage} — Fela</title>
```

Comment:
Fela being the static part, it’s better to put the dynamic part first so it’s always visible.